packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -105,6 +107,8 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
   outputs: TransactionRequestOutput[] = [];
   /** List of witnesses */
   witnesses: TransactionRequestWitness[] = [];
+  /** Base asset ID - should be fetched from the chain */
+  baseAssetId: string = ZeroBytes32;
```

Comment:
If we are going to have `baseAssetId` on `TransactionRequest` we need to make it an optional property, and this needs to be set later on.

Setting this early as `ZeroBytes32` can be misleading